working-with-openai/setting-up-openai-postman.md

```diff
@@ -0,0 +1,91 @@
+
+## Setting up OpenAI in Postman
+
+1. Open up an untitled POST request. Click the drop down for "No environment" and select "New Environment". Nothing will happen except what gets selected in the drop down.  
+![New Environment Drop Down in Postman](assets/setting-up-openai-postman-and-java/open-ai-postman-new-environment.png)  
+*Fig. Postman POST request with New Environment Selected*
+
+2. Add the following request to Postman: https://api.openai.com/v1/chat/completions.   
+We will talk more in the next lesson about what this request is and how it works, but for now, we will use it to verify that our environment is set up correctly. If you try and make the request now, you will get the following response body with an error that states you are missing an API key.  
+![OpenAI POST Response with Missing API Key Error](assets/setting-up-openai-postman-and-java/open-ai-postman-missing-api-key-error.png)  
+*Fig. OpenAI Postman response body with Missing API Key Error*
+
+3. Let's go ahead and add our authorization now. Open up the Authorization tab in Postman and you should see a pop up that asks if you need help authorizing the OpenAI API. Click "Set Up New Authorization"  
+![Postman Authorization Tab with New Authorization circled](assets/setting-up-openai-postman-and-java/open-ai-postman-set-up-new-authorization.png)  
+*Fig. OpenAI authorization tab with "Set Up New Authorization" circled*  
+
+4. You will be taken to a window to enter your OpenAI API key. Enter your API key and press "Save authorization". 
+![Tab to enter API key in Postman](assets/setting-up-openai-postman-and-java/open-ai-postman-enter-api-key.png)  
+*Fig. Tab to enter API key in Postman*
+
+5. Now let's check to see if our API key is registered! If you send the request now, you will see the following error.  
+![Open AI Postman Model Parameters Needed Error](assets/setting-up-openai-postman-and-java/open-ai-postman-model-parameters-needed-error.png)  
+*Fig. Model parameters needed error*  
+
+If you get this error, it's a pretty sure sign your API key is working. However, let's add a request body to make sure we are able to get a working response.  Go ahead and navigate to the body tab and select "raw" with JSON as the format. Copy and paste the following request body into the window:
+
+```py
+{
+    "model" : "gpt-3.5-turbo",
+    "messages" : [
+        {
+            "role" : "user",
+            "content" : "Write me a list of 5 flightless birds."
+        }
+    ]
+}
+```
+You should now be able to submit your POST request and get a response more in line with what you are looking for. Your response may look slightly different, but a sample of what you could see can be seen below:  
+
+![Successful API response](assets/setting-up-openai-postman-and-java/open-ai-postman-response-success.png) 
+*Fig. Successful OpenAI POST request with response body*  
```

Comment:
For images where the text may be small, we've started linking the full size image at the end of the caption. I think it's worth doing a quick review of the images to see if any others would benefit from a full image link.

```suggestion
*Fig. Successful OpenAI POST request with response body ([Full size image](assets/setting-up-openai-postman-and-java/open-ai-postman-response-success.png))*  
```